### PR TITLE
Show response string usage

### PR DESCRIPTION
**Pull request by dereuromark**
Improve example using new API
for https://github.com/cakephp/cakephp/pull/12932

### CODE REVIEW

**Comment by dereuromark** [comment]
any feedback?

File: en/development/testing.rst [hunk 1]
```diff
@@ -1195,13 +1195,18 @@ and make sure our web service is returning the proper response::
                 ['id' => 1, 'lng' => 66, 'lat' => 45],
             ];
             $expected = json_encode($expected, JSON_PRETTY_PRINT);
-            $this->assertEquals($expected, (string)$this->_response->getBody());
+            $this->assertEquals($expected, $this->_response->getStringBody());
```
Comment (markstory) [inline en/development/testing.rst]: Should we be telling folks to use `assertResponseEquals()` instead?

Comment (dereuromark) [reply en/development/testing.rst]: We could add that on top maybe? But some need the string itself for partial comparison or processing.

File: en/development/testing.rst [hunk 2]
```diff
@@ -1195,13 +1195,18 @@ and make sure our web service is returning the proper response::
...
         }
     }
 
 We use the ``JSON_PRETTY_PRINT`` option as CakePHP's built in JsonView will use
 that option when ``debug`` is enabled.
 
+.. versionadded:: 3.7.4
+    ``getStringBody()`` was added to read the rewound response.
```
Comment (markstory) [inline en/development/testing.rst]: Wouldn't this note be better placed in the response docs?

Comment (davidyell) [reply en/development/testing.rst]: What's a 'rewound' response?

Comment (cleptric) [reply en/development/testing.rst]: @davidyell Internally, the body is a `Stream`. To be able to read it, you have to first rewind it (set the pointer to the beginning).
You can simply typecast it to a string to archive the same result, as well.

**Pull request closed without merging**